test/membership_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+class MembershipTest < Minitest::Test
+  def start_date
+    '2015-11-27'
+  end
+
+  def end_date
+    '2015-12-31'
+  end
```

Comment:
Yep, I think they're a hangover from where I coped them from #2. Have removed them in 609e56f.